plugins/orchestrator/src/components/WorkflowEditor/WorkflowEditor.tsx

```diff
@@ -217,10 +217,10 @@ const RefForwardingWorkflowEditor: ForwardRefRenderFunction<
             if (canceled.get()) {
               return;
             }
-            const definition = fromWorkflowSource(source);
+            const definition = fromWorkflowSource(JSON.stringify(source.data));
```

Comment:
This is a workaround for the root cause, that the response should be a string
Better solve the root cause if possible